plugins/inputs/fibaro/fibaro.go

```diff
@@ -207,6 +208,12 @@ func (f *Fibaro) Gather(acc telegraf.Accumulator) error {
 			}
 		}
 
+		if device.Properties.Battery != nil {
+			if fValue, err := strconv.ParseFloat(device.Properties.Battery.(string), 64); err == nil {
```

Comment:
I see above that this is an existing issue, but would you be able to clean it up?  If we declare the struct type as a string we don't need to do this operation, and we avoid a panic if it is not a string.

Up on line 77:
```
Battery string `json:"batteryLevel"`
```

If you could do this for the other `interface{}` types it would be appreciated.

Reply:
Yes, I can do this. But I will need a little bit time, as I have to check out on how to test this on my fork. Unfortunately I am not so familliar with all the github features. Would I have to commit a new pull request, or can I work on the exisiting one, somehow? What is the normal way to do this? Sorry for bothering you with this.

Reply:
You can just make a new commit on your repository to this same branch, then when you push the branch to your remote it will automatically update this pull request.